runtime/redirector/redirector.c

```diff
@@ -775,18 +779,23 @@ JNI_CreateJavaVM(JavaVM **pvm, void **penv, void *vm_args)
 	memset(namedVM, 0, J9_VM_DIR_LENGTH);
 	chooseJVM(args, namedVM, J9_VM_DIR_LENGTH);
 
-	envOptions = getenv(ENVVAR_IBM_JAVA_OPTIONS);
-	if(NULL != envOptions && hasEnvOption(envOptions, "-locateVM")) {
+	envOptions = getenv(ENVVAR_OPENJ9_JAVA_OPTIONS);
+	if (NULL == envOptions) {
+		envOptions = getenv(ENVVAR_IBM_JAVA_OPTIONS);
+	}
+	if ((NULL != envOptions) && hasEnvOption(envOptions, "-locateVM")) {
 		J9StringBuffer *buffer = findDir(namedVM);
 		fprintf(stdout, "%s\n", jvmBufferData(buffer));
 		free(buffer);
 		exit(0);
-	} else for(i=0; i<args->nOptions; i++) {
-		if(0 == strcmp(args->options[i].optionString, "-locateVM")) {
-			J9StringBuffer *buffer = findDir(namedVM);
-			fprintf(stdout, "%s\n", jvmBufferData(buffer));
-			free(buffer);
-			exit(0);
+	} else {
+		for (i=0; i<args->nOptions; i++) {
```

Comment:
Since you've touched this line, can you fix the spacing?
`i<args->nOptions;` -> `i < args->nOptions;`

Reply:
Good plan.  Will do.